src/main/java/amazin/controller/BookController.java

```diff
@@ -97,15 +97,17 @@ public String deleteBook(@PathVariable("id") Long id, RedirectAttributes attribu
         return createRedirectViewPath(REQUEST_MAPPING_BOOK);
     }
 
-    @GetMapping("/searchByName")
+    @GetMapping("/search")
     public String searchByBookName(@RequestParam(value = "keywords", required = true) String text, Model model) {
 
         if (text.isEmpty())
             return createRedirectViewPath(REQUEST_MAPPING_BOOK);
 
+        String[] fieldNames = { "name", "description", "ISBN", "author", "publisher" };
```

Comment:
Can you create constants for those field names,consider adding them as static constant strings in Book class.

Reply:
done